source/authentication/apple.txt

```diff
@@ -59,379 +59,607 @@ following:
+the Apple authentication provider for *either* a mobile application or a 
+web application. If you would like to use Sign in with Apple for both, 
+you must use two separate App Services Apps.
 
-   .. step:: Create an App ID
+.. tabs::
 
-      An Apple **App ID** represents your application and allows you to
-      access services like Sign in with Apple. To configure the Apple ID
-      provider, you must create a new App ID.
+   .. tab:: Mobile Application
+      :tabid: mobile
 
-      1. Navigate to the :guilabel:`Certificates, Identifiers and Profiles`
-         page of the `Apple Developer Portal
-         <https://developer.apple.com/account>`_.
+      .. procedure::
 
-      #. Click :guilabel:`Identifiers` in the left navigation menu.
+         .. step:: Add Sign-in With Apple to Your App
 
-      #. Click the blue plus icon next to :guilabel:`Identifiers`.
+            Your app must have the **Sign-in with Apple** entitlement in 
+            order to use this capability. 
 
-      #. On the :guilabel:`Register a New Identifier` page, select
-         :guilabel:`App IDs` and then click :guilabel:`Continue`.
+            1. Select your app target in Xcode. 
+            
+            #. Go to the :guilabel:`Signing & Capabilities` tab, and 
+               select :guilabel:`+ Capability`.
 
-      #. On the :guilabel:`Register an App ID` page, select
-         the :guilabel:`Platform` that your app runs on and then enter a
-         brief :guilabel:`Description` and a :wikipedia:`reverse-dns
-         notation <Reverse_domain_name_notation>` :guilabel:`Bundle ID`.
+            #. Search for the **Sign in with Apple** capability, and select it.
+            
+            Then, confirm it is enabled in the Apple Developer Portal.
 
-      #. Scroll down the :guilabel:`Register an App ID` page until you see
-         the :guilabel:`Sign in with Apple` capability. Check the checkbox
-         to enable the capability.
+            1. Navigate to the :guilabel:`Certificates, Identifiers and Profiles`
+               page of the `Apple Developer Portal
+               <https://developer.apple.com/account>`_.
 
-      #. Press the :guilabel:`Continue` button at the top of the page. Complete
-         any other setup steps that apply to your app, and then press the
-         :guilabel:`Register` button.
+            #. Select the identifier for your app from the dropdown. This 
+               takes you to your :guilabel:`App ID Configuration` pane. 
 
+            #. Scroll down until you see the checkbox for :guilabel:`Sign In 
+               with Apple`. If that checkbox is not selected, select it. If
+               you've made changes, press the :guilabel:`Save` button.
 
-   .. step:: Create a Services ID
+            Note the :guilabel:`Bundle ID` of your app. You need the Bundle 
+            ID when you create a client secret and when you configure Sign in
+            with Apple in App Services.
```

Comment:
Where would you find the Bundle ID? Is it common enough that we can assume the reader knows?

Reply:
I think it's safe to assume that a dev who works with Xcode knows what a Bundle ID is and where to find it. Setting the Bundle ID is one of the first things you do when you create an App project in Xcode, and it stares at you every time you muck with the project details. The instructions about adding the Sign-in with Apple Capability assume you're working with Xcode. I have no idea how devs who do not work with Xcode manage this information.